src/tests/apps.cpp

```diff
@@ -2,41 +2,41 @@
 
 namespace marian {
 namespace bergamot {
-namespace testapp {
-
-// Utility function, common for all testapps.
-Response translateFromStdin(Ptr<Options> options, ResponseOptions responseOptions) {
-  // Prepare memories for bytearrays (including model, shortlist and vocabs)
-  MemoryBundle memoryBundle;
 
-  if (options->get<bool>("bytearray")) {
-    // Load legit values into bytearrays.
-    memoryBundle = getMemoryBundleFromConfig(options);
-  }
-
-  Service service(options, std::move(memoryBundle));
+namespace {
 
+std::string readFromStdin() {
```

Comment:
Ouch, convoluted to have so many copies but it is in a test.

Reply:
Carrying over to https://github.com/browsermt/bergamot-translator/issues/221, to be fixed later.